testing/apmsoak/Makefile

```diff
@@ -0,0 +1,34 @@
+APMSOAK_PATH ?= ../../systemtest/cmd/apmsoak
+APMSOAK_GOOS ?= linux
+APMSOAK_GOARCH ?= 386
+
+.default: all
+
+.PHONY: all
+all: $(SSH_KEY) terraform.tfvars init apmsoak apply
```

Comment:
I don't think there's any `SSH_KEY` target here, shouldn't we declare a default value for `SSH_KEY` which points to `~/.ssh/google_compute_engine`?